pyanaconda/iutil.py

```diff
@@ -193,7 +193,10 @@ def preexec():
             preexec_fn()
 
     with program_log_lock:
-        program_log.info("Running... %s", " ".join(argv))
+        if root:
```

Comment:
Seems like this will be almost always true as default value for root is '/' and in this case we don't do chroot?